src/components/form-editor/form-editor.component.tsx

```diff
@@ -143,7 +142,7 @@ const FormEditorContent: React.FC<TranslationFnProps> = ({ t }) => {
   };
 
   const inputDummySchema = useCallback(() => {
-    const dummySchema: FormSchema = {
+    const dummySchema: typeof schema = {
```

Comment:
@usamaidrsk Is this related to the bump?

Reply:
Looking again here, I think I will need to use exported types from the lib i.e , I had added that to fix some type errors after the bump.
Thanks

Reply:
I think its just that the type `label` was made optional in the form engine, and we have to update the interface `Schema` in here as well to make label optional, the changes are in PR #355 but it isn't merged in yet